test/cookbooks/firefox_package_test/recipes/default.rb

```diff
@@ -11,5 +11,18 @@
   end
 end
 
+# Copy an old version into a new version named directory to test
+# upgrading without having to worry about the paths.
+# There has to be a better way to copy directories... but it is not obvious.
+unless platform?('windows')
```

Comment:
FC023 deal specifically with string interpolation. We are using the PlatformIntrospection class here. Good eye though.